analyze_releases.py

```
@@ -0,0 +1,274 @@
+    # Generate summary stats
+    stats = generate_summary_stats(analysis)
+    
+    # Save detailed analysis
+    with open('classified_changes.json', 'w', encoding='utf-8') as f:
+        json.dump(analysis, f, indent=2, ensure_ascii=False)
+    
+    # Save summary stats
+    with open('analysis_summary.json', 'w', encoding='utf-8') as f:
+        json.dump(stats, f, indent=2, ensure_ascii=False)
+    
+    print(f"\nAnalysis complete!")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)